### ISSUE TITLE
Listings of places on the index page needs styled

### ISSUE THREAD

**Issue by travis-watson1** [VandyTeamProject/nativ#65]
Image for each place is too large and out of proportion.
Do we want to add more sections to the index page?

**Comment by christopheroye** [comment]
Yes. A few ideas in mind such as creating categories. What additional sections are you thinking about adding?

**Comment by travis-watson1** [comment]
> Yes. A few ideas in mind such as creating categories. What additional sections are you thinking about adding?

I was thinking about maybe adding a restaurant of the week? Maybe just pull a random restaurant from the DB? 

Maybe have a section for new restaurants? Top restaurants? 

Again, I don't know how hard this would be to implement. Just ideas.

**Comment by christopheroye** [comment]
That's a great idea! 

Jotting down user stories:

New & top restaurants - show new restaurants on the index page based on a timestamped order (general, location can vary). Also, when a user has identified their location a list of new restaurants only relevant to the specific location should appear. The top restaurant will be based on average traffic per 5-star rating and reviews

Restaurant of the week - this can be a combination of (Reviews & ratings, Favorites, Check-ins)

Check-in - Figure this can highlight this such as the user's status for the place via menu options (Status: Visited & Not-yet) and then on the user's profile page provide a toggle list to view places visited vs places not yet visited -- keeping tabs essentially.
